python/mujincontrollerclient/controllerclientbase.py

```diff
@@ -683,6 +679,19 @@ def FlushAndDownloadFile(self, filename, timeout=5):
             raise ControllerClientError(response.content.decode('utf-8'))
         return response
 
+    def FlushAndHeadFile(self, filename, timeout=5):
+        """downloads a file given filename
+
+        :return: a streaming response
+        """
+        response = self._webclient.Request('HEAD', '/file/download/', params={'filename': filename}, stream=True, timeout=timeout)
```

Comment:
The comment of this function is outdated.

Are you sure you need `stream=True`? I don't think you need that.

Reply:
That's quick. Fixed